omou-frontend/src/reducers/calenderReducer.js

```diff
@@ -0,0 +1,44 @@
+import initialState from './initialState';
+import * as actions from "../actions/actionTypes"
+
+export default function calender(state = initialState.CalenderData, { payload, type }) {
+    let newState = state;
+    let obj = newState.events_in_view;
+    switch (type) {
+        case actions.ADD_EVENT:
+            newState.events_in_view.push(payload);
+            console.log('CalenderData Action', payload, newState);
+            return state;
+
+        case actions.DELETE_EVENT:
+            for (let i = 0; i < newState.events_in_view.length; i++) {
```

Comment:
If you're iterating through each object in `newState.events_in_view`, uses `newState.events_in_view.forEach( (event) => { //function body }` . Check out [documentation here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach)